<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="dev_select_clicked">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dev_select">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Fake AP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="show_scan_ap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>691</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>691</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ap_scanner_clicked">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>AP Scanner</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
